src/popup/views/Loading/index.tsx

```diff
@@ -12,7 +10,6 @@ const LoadingEl = styled.div`
 
 export const Loading = () => (
   <LoadingEl>
-    <FullscreenStyle />
```

Comment:
sorry @jeesunikim ! looks like this breaks the logic that opens a new window for <Welcome />
https://github.com/stellar/lyra/blob/master/src/popup/Router.tsx#L100

your PR had some good fixes for <Loading> that we can keep tho!